apps/cde-ui/src/app/pages/create-visualization-page/create-visualization-page.component.html

```diff
@@ -0,0 +1,205 @@
+
+  <mat-card class="creation-step data">
+    <mat-card-header>
+      <mat-card-title>
+        <div class="step-number">1</div>
+        Format and Upload Data
+      </mat-card-title>
+    </mat-card-header>
+    <mat-card-actions>
+      <div class="left-side">
+        <div class="column-content">
+          <h2 class="subtitle">Upload Cell Type Table</h2>
+          <div class="column-info">
+            <div class="column-info-column">
+              <span class="column-info-header">Required Columns</span>
+              <span class="column-info-list">X, Y, and Cell Type</span>
+            </div>
+            <div class="column-info-column">
+              <span class="column-info-header">Optional Columns</span>
+              <span class="column-info-list">Z and Cell Ontology ID</span>
+            </div>
+          </div>
+          @if (dataUploaded) {
+            <div class="column-info-column">
+              <span class="column-info-header">File loaded:</span>
+              <span class="column-info-list">(file name)</span>
+            </div>
+            <button mat-button (click)="removeCSV()">Remove CSV</button>
+          } @else {
+            <button mat-button (click)="upload()">Upload CSV</button>
+            <input
+              #fileInput
+              (change)="handleFile($event)"
+              type="file"
+              accept="text/csv"
+              name="fileInput"
+              id="fileInput"
+              style="display: none"
+            />
+          }
+        </div>
+      </div>
+      <div class="right-side">
+        <div class="column-content">
+          <h2 class="subtitle">Cell Type Table Template</h2>
+          <button mat-button>Use Template -></button>
+        </div>
+      </div>
+    </mat-card-actions>
+  </mat-card>
+
+  <mat-card class="creation-step anchor">
+    <mat-card-header>
+      <mat-card-title>
+        <div class="step-number">2</div>
+        Optional: Select Anchor Cell type
+      </mat-card-title>
+    </mat-card-header>
+    <mat-card-actions>
+      <mat-form-field>
+        <mat-select [value]="visualizationForm.value.anchorCellType" formControlName="anchorCellType">
+          @for (cell of cellTypes; track cell) {
+            <mat-option [value]="cell.value">{{ cell.viewValue }}</mat-option>
+          }
+        </mat-select>
+      </mat-form-field>
+    </mat-card-actions>
+  </mat-card>
+
+  <mat-card class="creation-step metadata" formGroupName="metadata">
+    <mat-card-header>
+      <mat-card-title>
+        <div class="step-number">3</div>
+        Add metadata
+      </mat-card-title>
+    </mat-card-header>
+    <mat-card-actions>
+      <div class="row-1">
+        <div class="metadata-option">
+          <span>Visualization Title</span>
+          <mat-form-field>
+            <input
+              matInput
+              placeholder="Title"
+              [value]="visualizationForm.value.metadata ? visualizationForm.value.metadata.title : ''"
```

Comment:
Don't these automatically sync with the form group without having to specify `[value]="..."`?